From 637d06bb3e7e7c03456dd97d0318d005eef0a9a4 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Thu, 8 Mar 2018 20:05:50 +0800
Subject: [PATCH] drm/rockchip: rgb: add support rk3308 rgb

add support for rgb panel or rgb covert to other interface panel.

Change-Id: I190ce6e08d38f794ecabb863e0def5e74890f75a
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 .../devicetree/bindings/display/rockchip/rockchip-rgb.txt       | 1 +
 drivers/gpu/drm/rockchip/rockchip_rgb.c                         | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt b/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt
index ab121f20f0bd..99b27f73e1a2 100644
--- a/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt
+++ b/Documentation/devicetree/bindings/display/rockchip/rockchip-rgb.txt
@@ -5,6 +5,7 @@ Required properties:
 - compatible: matching the soc type:
 	- "rockchip,rv1108-rgb";
 	- "rockchip,rk3066-rgb";
+	- "rockchip,rk3308-rgb";
 
 Optional properties:
 - pinctrl-names: must contain a "lcdc" entry.
diff --git a/drivers/gpu/drm/rockchip/rockchip_rgb.c b/drivers/gpu/drm/rockchip/rockchip_rgb.c
index 5b411f8a53e9..09deb92e772d 100644
--- a/drivers/gpu/drm/rockchip/rockchip_rgb.c
+++ b/drivers/gpu/drm/rockchip/rockchip_rgb.c
@@ -149,6 +149,8 @@ static const struct of_device_id rockchip_rgb_dt_ids[] = {
 		.compatible = "rockchip,rv1108-rgb",
 	}, {
 		.compatible = "rockchip,rk3066-rgb",
+	}, {
+		.compatible = "rockchip,rk3308-rgb",
 	},
 	{}
 };
-- 
2.35.3

